packages/react/src/__tests__/ssr.test.tsx

```diff
@@ -1,7 +1,9 @@
 /**
  * @jest-environment node
  */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
nit
You could also do
`/* eslint-disable import/no-extraneous-dependencies */`
import { styled } from '@compiled/react';
import { CC as CompiledRoot } from '@compiled/react/runtime';
`/* eslint-enable import/no-extraneous-dependencies */`

Reply:
thanks! that's really neat.